### PR TITLE
[MKT_481]: feat/ Translations in every language for sustainability page

### PR DESCRIPTION
**Pull request by jaaaaavier**
Here are the transaltions for every language for the sustainability page.
Also the path to the new LP are added to the footer and navbar

### CODE REVIEW

File: src/assets/lang/es/sustainability.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      }
+    }
+  },
+  "FeatureSection": {
+    "title": "Cómo nos comprometemos con un futuro sostenible",
+    "description": "Al promover iniciativas ecológicas, nuestro objetivo es proteger el planeta y esforzarnos por lograr una solución de almacenamiento en la nube más limpia y ecológica para 2025.",
+    "cards": {
+      "reduceCarbonEnergy": {
+        "number": "1",
+        "title": "Reduce carbon energy",
```
Comment (larryrider) [inline src/assets/lang/es/sustainability.json]: This ones are in english and not in spanish

File: src/assets/lang/ru/metatags-descriptions.json [hunk 2]
```diff
@@ -163,5 +163,10 @@
     "id": "cloud-object-storage",
     "title": "Internxt — Объектное Хранилище",
     "description": "Internxt предоставляет частное и доступное объектное хранилище для безопасного хранения данных с использованием протокола S3 для эффективного управления вашими потребностями в хранении."
+  },
+  {
+    "id": "green-cloud-computing",
+    "title": "Archiviazione cloud verde e sostenibile",
+    "description": "I data center ecologici di Internxt promuovono il nostro impegno per la sostenibilità e l'informatica cloud ecologica, supportando un ambiente più verde e sostenibile."
```
Comment (larryrider) [inline src/assets/lang/ru/metatags-descriptions.json]: This one is in italian and not in russian

**Review by larryrider — CHANGES_REQUESTED** [review]
The are also 2 files missing from the "pt-br" language ("metatags-descriptions.json" & "sustainability.json").. Is that how it should be?

File: src/assets/lang/es/sustainability.json (hunk 1 above)
Comment (jaaaaavier) [reply src/assets/lang/es/sustainability.json]: Updated

File: src/assets/lang/ru/metatags-descriptions.json (hunk 2 above)
Comment (jaaaaavier) [reply src/assets/lang/ru/metatags-descriptions.json]: Done

**Pull request merged**